nemo/collections/asr/parts/submodules/rnnt_greedy_decoding.py

```diff
@@ -2819,7 +2823,7 @@ def forward(
 
         return (packed_result,)
 
-    def _greedy_decode_blank_as_pad(
+    def _greedy_decode_blank_as_pad_loop_frames(
```

Comment:
I am thinking maybe we could just delete this loop-frame version of decoding since it only works with TDT models trained with omega != 0.0; otherwise we will be having a lot of insertion errors.

Reply:
Yes that sounds good.